src/packages/dwlib/dwlib.cc

```diff
@@ -42,6 +45,9 @@ void query_multiple_short(svalue_t *arg, const char *type, int no_dollars, int q
   object_t *ob;
   char *fun;
 
+  //mdw--old code dern't work, copying this from elsewhere.
+  auto max_string_length = CONFIG_INT(__MAX_STRING_LENGTH__);
```

Comment:
also i made a fringe repository that excludes um... driver files and stuff
and just has the lib. (maybe... on second thought... you might actually
/want/ the driver stuff

essentially what i'm working on is just:
http://lpmuds.net/files/discworld/dw_fluffos_v3.zip
unzipped and fluff compiled. as i do whatevs i'll git commit etc et al?

On Sun, May 31, 2015 at 8:15 PM, Mike <brainfacemike@gmail.com> wrote:

> That's actually a variable set in some other config file that the DW lib
> needed to reference--i assume we can change it affecting the whole lib but
> there may also be c limitations as you say? (And if so we could work around
> it by splitting things up and such?)
> On May 31, 2015 6:41 PM, "FractalBobz" <notifications@github.com> wrote:
>
>> In src/packages/dwlib/dwlib.cc
>> <https://github.com/FractalBobz/fluffos/pull/1#discussion_r31395103>:
>>
>> > @@ -42,6 +45,9 @@ void query_multiple_short(svalue_t *arg, const char *type, int no_dollars, int q
>> >    object_t *ob;
>> >    char *fun;
>> >
>> > +  //mdw--old code dern't work, copying this from elsewhere.
>> > +  auto max_string_length = CONFIG_INT(__MAX_STRING_LENGTH__);
>>
>> I'm assuming by changing MAX STRING LENGTH we can change the max length
>> of posts?
>> Or is this a base C variable?
>> I'll look into this as kitty wants bigger posts.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/FractalBobz/fluffos/pull/1/files#r31395103>.
>>
>